gst/nnstreamer/elements/gsttensor_converter.c

```diff
@@ -1158,6 +1158,7 @@ gst_tensor_converter_chain (GstPad * pad, GstObject * parent, GstBuffer * buf)
     case _NNS_OCTET:
       if (gst_tensors_config_is_flexible (config)) {
         /* update dimension with buffer size */
+        g_assert (buf_size <= G_MAXUINT32);
```

Comment:
buf_size is gsize.
```
typedef unsigned int gsize;
```

This looks meaningless.

Reply:
yes, but svace tool detects size as long :(

Reply:
Then, please mark it "false positive" and report SE.